### ISSUE TITLE
Loading issue with rspec + ActiveRecord (without rails)

### ISSUE THREAD

**Issue by adamgotterer** [jeremyevans/fixture_dependencies#27]
I have a non rails project that I'm trying to load the library with. I followed the instructions for "other testing libraries". But when I'm having trouble loading the helper methods. To demonstrate I try the require in pry.

```ruby
[6] pry(main)> require 'fixture_dependencies/helper_methods'
LoadError: cannot load such file -- fixture_dependencies/helper_methods
from (pry):6:in `require'
```

I also tried:

```ruby
[7] pry(main)> require 'fixture_dependencies'
LoadError: cannot load such file -- sequel/extensions/inflector
from /gems/gems/fixture_dependencies-1.9.0/lib/fixture_dependencies.rb:1:in `require'
```

I'm guessing it needs to be aware that I'm using AR. So I tried to do this first:

```ruby

[8] pry(main)> require 'fixture_dependencies/active_record'
NameError: uninitialized constant FixtureDependencies
from /gems/gems/fixture_dependencies-1.9.0/lib/fixture_dependencies/active_record.rb:1:in `<top (required)>'
```

**Comment by jeremyevans** [comment]
For case 1, `fixture_dependencies/helper_methods` doesn't exist.  That's not listed anywhere in the documentation, so it isn't expected to work.

For case 2, it looks like active_support hasn't been loaded yet.  FixtureDependencies requires some inflection methods on String, and if they aren't already loaded, tries loading them using Sequel's extension.  You should probably require the active_support inflection support if you want to use that before requiring `fixture_dependencies`.

For case 3, you shouldn't require `fixture_dependencies/active_record` manually, FixtureDependencies  will require it if ActiveRecord::Base is defined after the core support is loaded.

To sum up, require active_record and active_support inflection support first, then require fixture_dependendencies, then things should work.  If things don't work when you do that, please post a minimal self contained example and I will look into it.

**Comment by adamgotterer** [comment]
Thanks. I got it working by adding the inflector from activesupport.

FWIW the readme (https://github.com/jeremyevans/fixture_dependencies#with-other-testing-libraries) says to use `fixture_dependencies/helper_methods`. But after digging in some more I realized that it's not part of the 1.9 release and appears to be part of an uncut version.

Thanks again! Looking forward to trying this out.

**Comment by jeremyevans** [comment]
Sorry about that, I was looking at a local checkout that is behind master and forgot about the addition of `helper_methods`.  I'll try to put out a new release soon so that the README matches the current release.
